e2e/nx-go-e2e/tests/nx-go.spec.ts

```diff
@@ -7,14 +8,26 @@ import {
   runNxCommandAsync,
   uniq,
   updateFile,
-} from '@nrwl/nx-plugin/testing'
+} from '@nx/plugin/testing'
 import { join } from 'path'
 
+const ensureProjectAndLayout = () => {
+  ensureNxProject('@nx-go/nx-go', 'dist/packages/nx-go')
+  const nxJson = readJson<NxJsonConfiguration>('nx.json')
+  nxJson.workspaceLayout = {
```

Comment:
Since these tests expect the `'app-and-libs'` layout, specifying the `appDir` and `libsDir` was necessary since `ensureNxProject` no longer produces a workspaces with this layout by default.